nil/services/nilservice/service.go

```diff
@@ -257,10 +257,12 @@ type syncersResult struct {
 	funcs   []concurrent.FuncWithSource
 	syncers []*collate.Syncer
 	wgInit  sync.WaitGroup
```

Comment:
Please use errgroup

https://pkg.go.dev/golang.org/x/sync/errgroup

Reply:
I don't understand exactly how you propose to apply it here. Where do I call `errgroup.Go`?